.github/workflows/build.yml

```diff
@@ -118,7 +118,7 @@ jobs:
     env:
       ARCH: arm
       LIBC: glibc
-      DOCKER_BUILDER: rochdev/holy-node-box:12-arm32v7
+      DOCKER_BUILDER: node:20
```

Comment:
This will break support for old GLIBC versions.

Reply:
(I guess there should probably be a test for that but it looks like there isn't 😅)